src/super_gradients/training/utils/tensor_formats/default_formats.py

```diff
@@ -0,0 +1,103 @@
+from super_gradients.common.object_names import ConcatenatedTensorFormats
+from super_gradients.training.utils.tensor_formats.formats import ConcatenatedTensorFormat, BoundingBoxesTensorSliceItem, TensorSliceItem
+from super_gradients.training.utils.tensor_formats.bbox_formats import (
+    XYXYCoordinateFormat,
+    XYWHCoordinateFormat,
+    CXCYWHCoordinateFormat,
+    NormalizedXYXYCoordinateFormat,
+    NormalizedXYWHCoordinateFormat,
+    NormalizedCXCYWHCoordinateFormat,
+)
+
+XYXY_LABEL = ConcatenatedTensorFormat(
```

Comment:
So what I don't like about current factory is that one has to make changes in three places to add new format (or optimizer, or model or loss):

1) Add new line to ConcatenatedTensorFormats: `MY_NEW_XYXY_LABEL_FORMAT = "MY_NEW_XYXY_LABEL_FORMAT"`
2) Define the format itself: `MY_NEW_XYXY_LABEL_FORMAT = ConcatenatedTensorFormat(...)`
3) Register it in `DEFAULT_CONCATENATED_TENSOR_FORMATS`

This is unnecessary self-repeating process which we can avoid by using decorators or function calls:

```
register(BBoxFormatFactory, "XYWH_LABEL", ConcatenatedTensorFormat(
    layout=(
        BoundingBoxesTensorSliceItem(name="bboxes", format=XYWHCoordinateFormat()),
        TensorSliceItem(length=1, name="labels"),
    )
))
```
By doing so we ditch DEFAULT_CONCATENATED_TENSOR_FORMATS and ConcatenatedTensorFormats classes and fill our factories in one place

Reply:
I agree that a lot of unnecessary self-repeating, but it changes the current logic done for optimizer, model, loss, ...
If we change this, I think we should change it for all the factories to keep a homogenous approach.

Your proposition in the case of classes, like a model for instance, would be working with a decorator, the same way we ask people with a custom models to do, right?

@register_model
class Resnet(...):
    ...

Reply:
Your proposition for classes would be working with a decorator, the same way we ask people with a custom models to do, right?

```
@register_model
class Resnet(...):
    ...
```

Reply:
Whether it is possible - yes. For cases where we cannot do this (as with example of tensor format layout) - through `register_WHATEVER(...)` call.
Agree this is a huge change and def. not part of this PR